Add Bootstrap info

saracook opened the pull request:
https://osc.github.io/ood-documentation/latest/customizations.html#overriding-pages

I had to dig to find the Bootstrap version because it's not documented, so this is an attempt to fix that for future readers.

It's worth explicitly telling people what Bootstrap is and that it is built into OOD, if only to improve the search results.

Currently, a search for 'Bootstrap' in the docs produces outdated release notes that reference previous versions of Bootstrap, and an unexplained reference to Bootstrap alerts in https://osc.github.io/ood-documentation/latest/customizations.html#announcements. Users should be able to get better information for that search.

I'm not claiming that my text is perfect; please wordsmith as needed.

Thanks!
Sara

saracook committed:
Add Bootstrap info

I had to dig to find the Bootstrap version because it's not documented.  

It's worth explicitly telling people what Bootstrap is and that it is built into OOD.

johrstrom commented on source/customizations/overriding-partials.inc:
```diff
@@ -34,8 +34,10 @@ distributed by the Open OnDemand package.
 .. code-block:: html
 
-  <div class="text-center h-5 my-3" >
+  <div class="text-center p-4 my-3" >
```
Can we keep `h5` (and also correct it)?

saracook replied:
Oh, that was supposed to be h5! That makes so much more sense than trying to set height to 5, which would be the closest Bootstrap meaning. I was so confused.

Text alignment, padding, and margins are better examples than the header utility class. The header utility classes are a more niche thing that most people won't need to use in their apps.

johrstrom replied:
> Text alignment, padding, and margins are better examples than the header utility class. The header utility classes are a more niche thing that most people won't need to use in their apps.

OnDemand (or maybe just me) uses it a lot to change font sizes. Though looking through the docs there's an actual class `fs-N` for the same. In any case, that was what I was trying to illustrate - how to change the font size. 

Can we add `fs-5` then?

saracook replied:
Perfect! Change is made.

johrstrom commented:
Can you rebase this to `latest` instead of `develop`. `latest` is the current content people see, and develop is for the _next_ version whatever that may be. In any case, I'd say this applies to the current release, so I'd like to put it into latest.

saracook committed:
Enhance example and update Bootstrap description to match

Updated the footer's font size and improved the description to include font size utility classes.

johrstrom reviewed (approved):
Thanks for the update!
